README.md

```diff
@@ -37,23 +49,35 @@ Example payloads can be found in [files directory](files)
 
 ## Requirements
 
+You are welcome to use whatever amount of time you see fit on this assessment, but we do not expect
+developers to spend more than two hours implementing both parts.
+
 ### Part 1
 
-* The application should accept JSON payloads via a route
-    * When the application receives a new payload it must be validated (reference schema), parsed and (if the data is correct) stored in the database
+* The application should accept JSON payloads via a route.
```

Comment:
I know we are using the term `JSON payload` since object data is somewhat loosely structured but it's probably better just to call this a plane old API request.

```
* The application should accept JSON requests that match the reference schema.
* Valid requests should upsert each objects data in the database.
* Invalid requests should return an appropriate validation response.
```

What do you think?

Reply:
Works for me. Payload does kind of imply something more complex than just a request